src/main/java/com/arondor/arender/alfresco/content/transformer/ContentTransformARenderRestClient.java

```diff
@@ -0,0 +1,126 @@
+package com.arondor.arender.alfresco.content.transformer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.logging.Logger;
+
+import org.springframework.core.io.Resource;
+import org.springframework.http.*;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
+import org.springframework.util.StringUtils;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+public class ContentTransformARenderRestClient
+{
+    private static final Logger LOGGER = Logger.getLogger(ContentTransformARenderRestClient.class.getName());
+
+    private String address;
+
+    private final RestTemplate template;
+
+    // upgrade transfer partial size, 8MB at a time
+    private static final int partialTransferBlockSize = 8 << 20;
+
+    public ContentTransformARenderRestClient()
+    {
+        template = new RestTemplate();
+        SimpleClientHttpRequestFactory clientHttpRequestFactory = new SimpleClientHttpRequestFactory();
+        clientHttpRequestFactory.setConnectTimeout(120000);
+        clientHttpRequestFactory.setReadTimeout(120000);
+        template.setRequestFactory(clientHttpRequestFactory);
+    }
+
+    public float getWeatherPerformance()
+    {
+        try
+        {
+            return template.getForObject(UriComponentsBuilder.fromHttpUrl(address + "weather").build().toUri(),
+                    Float.class);
+        }
+        catch (Exception e)
+        {
+            LOGGER.info("Caught exception while trying to fetch weather score, is rendition server down?");
```

Comment:
Add the exception that was thrown here to the logged information.

That way, if something goes wrong, it's easier to debug the problem.

```suggestion
            LOGGER.log(java.util.logging.Level.INFO, "Caught exception while trying to fetch weather score, is rendition server down?", e);
```

I know this isn't your code, but I think this change would make debugging an issue here a lot easier.